internal/oci/container.go

```diff
@@ -462,6 +462,9 @@ func (c *Container) pid() (int, error) {
 	if err := c.verifyPid(); err != nil {
 		return 0, err
 	}
+	if err := unix.Kill(c.state.InitPid, 0); errors.Is(err, unix.ESRCH) {
```

Comment:
Also, since it's `unix`, the error is bare, and it would be a _tad_ faster to do a direct comparison, ie `err == unix.ESRCH`.

Reply:
updated